deps/wazuh_testing/wazuh_testing/tools/system.py

```diff
@@ -244,3 +244,13 @@ def run_shell(self, host: str, cmd: str, check: bool = False):
             stdout (str): The output of the command execution.
         """
         return self.get_host(host).ansible('shell', cmd, check=check)['stdout']
+
+
+def clean_environment(host_manager, target_files):
+    """Clears a series of files on target hosts managed by a host manager
+    Args:
+        host_manager (object): a host manager object with not None inventory_path
+        target_files (dict): a dictionary of tuples, each with the host and the path of the file to clear.
```

Comment:
```suggestion
        target_files (dict): a list of tuples, each with the host and the path of the file to clear.
```

Reply:
Done in commit [0f4c9](https://github.com/wazuh/wazuh-qa/pull/2443/commits/0f4c91272c4266608f4ecae865da522ea77d24c6)